webapp/src/modules/location/selectors.js

```diff
@@ -7,3 +7,21 @@ export const getParams = state => getState(state).params
 
 export const isStaticPage = state =>
   STATIC_PAGES.includes(state.router && state.router.location.pathname)
+
+export const isModalPage = state => {
+  if (state.router && state.router.location.pathname) {
+    const lastPartOfUrl = state.router.location.pathname.split('/').pop()
+    switch (lastPartOfUrl) {
+      case 'edit':
+      case 'buy':
+      case 'sell':
+      case 'cancel-sale':
+      case 'transfer':
+      case 'settings':
+        return true
```

Comment:
I might be crazy, but can we try to use [matchParam](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/matchPath.md) with our `locations` object. What do you think?